### ISSUE TITLE
Extensions published as inactive

### ISSUE THREAD

**Issue by kineticsquid** [EclipseFdn/open-vsx.org#2000]
Several users have opened issues recently regarding published extensions, both new and new versions, being published as inactive. I'd like to bring those discussions together here.
- https://github.com/EclipseFdn/open-vsx.org/issues/1996
- https://github.com/EclipseFdn/open-vsx.org/issues/1977
- https://github.com/EclipseFdn/open-vsx.org/issues/1946
- https://github.com/EclipseFdn/open-vsx.org/issues/1991
- https://github.com/EclipseFdn/open-vsx.org/issues/1990

I experimented with several of these by cloning the repos, changing `name`, `displayName`, `version` and `publisher`. I then ran `vsce package` to package and `npx ovsx publish -p ****** --debug` to publish to my namespace `kineticsquid`. I was able to publish the four extensions I tried.

One note, both the UI and the `ovsx publish` command return while asynchronous file processing can still be going on. If this is the case, the extension will be marked as inactive. Once the file processing completes, assuming no errors, the extensions will be marked active and will show up. On a couple of occasions the extensions showed up as inactive for 10 or 15 seconds before becoming active. 

I did see one failure resulting in an inactive extension that I was not able to reproduce. This is what `--debug` returned:
```
Error: Internal Server Error
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/ovsx/lib/registry.js:176:40)
    at IncomingMessage.emit (node:events:538:35)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
```

Two things here. First, is there a bug in the publishing processing? Second though, we need to think about better feedback in the publishing process.

**Comment by kineticsquid** [comment]
FYI @sjsepan3 @amvanbaren @4lex4 @Release-Candidate @andiwang1993

**Comment by Release-Candidate** [comment]
Thanks for your response.

As I've said, after I did try to publish a really big extension (45MB, because my local `.yarn` directory has been accidentally included) I could neither publish a new version of this extension nor new versions of my other two extensions afterwards. But the directory which caused the problem has never been checked in to Git, so the packages produced by a git checkout are 1/10th of the size of the erroneous package and should cause no problems.

So if you want to reproduce the error, you may want to try to publish the version 0.8.0 - Git tag [v0.8.0](https://github.com/Release-Candidate/vscode-ocaml-alcotest-test-adapter/tree/v0.8.0) - or 0.8.1 - Git tag [v0.8.1](https://github.com/Release-Candidate/vscode-ocaml-alcotest-test-adapter/tree/v0.8.1) and add 40MB of random data to a `.yarn` directory in the project root before building the extension.

**Comment by amvanbaren** [comment]
> First, is there a bug in the publishing processing?

There was. The release today seems to have fixed it.

**Comment by sjsepan3** [comment]
@kineticsquid 
I was able to publish today. I don't know if it was from clearing out the de-activated older attempts (likely) or a bug-fix (possible).

**Comment by amvanbaren** [comment]
@Release-Candidate Did you bump the version and republish?

**Comment by Release-Candidate** [comment]
@amvanbaren Thanks, I could publish new versions of all 3.
The older versions that didn't get published are lost forever, right?

**Comment by kineticsquid** [comment]
@Release-Candidate for `vscode-ocaml-alcotest-test-adapter` on the admin panel, I can see releases `0.5.0, 0.6.0, 0.7.0, 0.8.0, 0.8.1, 0.8.2, 0.8.3`. The final 4 are missing from the extension page, likely because they were deactivated. If you want that history, now that we've solved this publishing issue, I believe I can un-publish `0.8.0 - 0.8.3` and you could re-publish.

**Comment by Release-Candidate** [comment]
@kineticsquid no, thanks, that would be too much work on your side for something that isn't really needed anyways. The question had just been out of curiosity.
